src/modules/modmail.js

```diff
@@ -0,0 +1,68 @@
+const Eris = require("eris");
+const config = require("../config");
+const utils = require("../utils/utils");
+const snippets = require("../data/snippets");
+
+/**
+ * @param {Eris.CommandClient} bot
+ */
+module.exports = bot => {
+
+  /**
+   * @returns {Promise<void>}
+   */
+  async function disableModmail() {
+    await snippets.del("isDisabled");
+    return await snippets.add("isDisabled", "disabled", true, bot.user.id);
+  }
+
+  /**
+   * @returns {Promise<void>}
+   */
+  async function enableModmail() {
+    await snippets.del("isDisabled");
+    return await snippets.add("isDisabled", "enabled", true, bot.user.id);
+  }
+
+  bot.registerCommand("modmail", async (msg, args) => {
+    if (! utils.isAdmin(msg.member) || ! msg.member.roles.includes("987377218927861760")) return;
+    const snippet = await snippets.get("isDisabled");
+    if (! snippet) await snippets.add("isDisabled", "enabled", true, bot.user.id);
```

Comment:
i *really* don't like how this is being handled. for now it's fine but really we should find a proper way to store the value

Reply:
I don't either however the snippet is uneditable so there's no risk of accidental edit or deletion by staff.